app/mailers/verification_mailer.rb

```diff
@@ -5,7 +5,7 @@ class VerificationMailer < ApplicationMailer
 
   def account_ownership_verification_email
     @user = User.find(params[:user_id])
-    email_authorization = EmailAuthorization.create(user: @user, type_of: "account_ownership")
+    email_authorization = EmailAuthorization.create!(user: @user, type_of: "account_ownership")
```

Comment:
Caught this little bug in the process, as the mailer should fail if something goes wrong here